packages/cli/templates/default-backend-plugin/src/dynamic/alpha.ts.hbs

```diff
@@ -0,0 +1,8 @@
+import { BackendDynamicPluginInstaller } from '@backstage/backend-plugin-manager';
+
+import { {{id}}Plugin } from '../service/router';
```

Comment:
```suggestion
import { {{id}}Plugin } from '../plugin';
```

Reply:
There is no `../plugin` file in `default-backend-plugin`.

Reply:
Same as in the `default-backend-module` isn't it: https://github.com/janus-idp/backstage-plugins/pull/782/files#diff-a1c7ed3013180570da88302938a6eb7c30d0cde2568ef89aa6ce791da224d3b6 ?

Reply:
Not sure what you're referring to.

[`default-backend-module`](https://github.com/tumido/backstage-plugins/tree/update-new-plugin-tooling/packages/cli/templates/default-backend-module) has `src/module.ts` with the actual module code, therefore it imports `../module` in `src/dynamic/alpha.ts`

[`default-backend-plugin`](https://github.com/tumido/backstage-plugins/tree/update-new-plugin-tooling/packages/cli/templates/default-backend-plugin) has `src/service/router.ts` with the actual module integration, therefore it imports `../service/router` in `src/dynamic/alpha.ts`

Neither of those templates have `src/plugin.ts`, that file is traditionally used in frontend plugins only. None of the backend plugin templates has this file, here or upstream. 🤷